LondonBritishLibrary/orient/BLorient12859.xml

```diff
@@ -0,0 +1,437 @@
+                                ለተጽእሮ። ፈረስከ፡ ዘትፄዓኖ፡ እንደ፡ ያስተቃንጽ፡ ዕግሮ። ለረዊደ፡ ጸርየ፡ ይሩጽ፡ ወኢያንትግ፡ በጽሮ።</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <title ref="LIT7120Respect"/>
+                            <note>Written in the right column under the fifth picture and in the left and middle column under the sixth picture.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ዑን፡ ነቢ፡ ሰላማ፡ <gap reason="ellipsis"/> ከማሁ፡ አሸብሮሙ፡ ለሰብአ፡ ኵሉ፡ ፍጥረት፡
+                                በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i7">
+                            <title ref="LIT5888AsmatPrayer">Prayer for gaining respect from Gāllā, Šānqǝllā and all kinds of people</title>
+                            <note>Written in the right column under the sixth picture and in the three columns under the seventh picture. The text is 
+                                partly similar to the prayer above, but using several 
+                                <term key="Magic">magic words (<foreign xml:lang="gez">ʾasmāt</foreign>)</term> at the beginning. 
+                                The letter <foreign xml:lang="gez">ዥ</foreign> is represented by the more ancient or irregular form with four little 
+                                strokes marking the sixth order: <figure><graphic url="https://betamasaheft.eu/resources/images/snippets/BLorient12996/sign2.png" height="30px">
+                                <desc>Symbol as represented in the catalogue by Stefan Strelcyn</desc></graphic></figure>.</note>
+                            <incipit xml:lang="gez">ዥኖ፡ ከሚጽ፡ ከደቢል፡ እልሚል፡ <gap reason="ellipsis"/> ከማሁ፡ አሸብሮሙ፡ ለሰብአ፡ ጋላ፡ ወለሰብአ፡ ሻንቅላ፡ ወለሰብአ፡
+                                ኵሉ፡ ፍጥረት፡ በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i8">
+                            <title ref="LIT5888AsmatPrayer">Prayer for gaining respect from Gāllā, Šānqǝllā and all kinds people</title>
+                            <note>Written in the left and middle columns under the eighth picture. The text is partly similar to the two prayers above, 
+                                but using different <term key="Magic">magic words (<foreign xml:lang="gez">ʾasmāt</foreign>)</term> at the 
+                                beginning.</note>
+                            <incipit xml:lang="gez">በስመ፡ ሽሐብር፡ ስለከላሂምሸርታ፡ <gap reason="ellipsis"/> በኃይለ፡ ዝንቱ፡ አስማቲከ፡ 
+                                አሸ<supplied reason="undefined" resp="PRS8999Strelcyn"></supplied>ብብሮሙ፡ ለሰብአ፡ ጋላ፡ ወለሻንቅላ፡ ወለሰብአ፡ ኵሉ፡ ፍጥረት፡ 
```

Comment:
The ብ should in inside `<supplied reason="undefined" resp="PRS8999Strelcyn"></supplied>`, right?

I also wonder if the symbol that Strelcyn uses might rather be his was of showing `<surplus>`…